### PR TITLE
librepcb: 0.1.4 -> 0.1.5

### PR DESCRIPTION
**Pull request by Luz**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ x ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ x ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ x ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/science/electronics/librepcb/default.nix [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
-  version = "0.1.4";
+  version = "0.1.5";
 
   src = fetchFromGitHub {
     owner = "LibrePCB";
     repo = "LibrePCB";
     fetchSubmodules = true;
-    rev = "ae04eef5a71b5ba66ae2cee6b631c1c933ace535";
-    sha256 = "0wk5qny1jb6n4mwyyrs7syir3hmwxlwazcd80bpxharmsj7p0rzc";
+    rev = "a435c5095c6f94df4da130fe832ea97f42d17901";
```
Comment (prusnak) [inline pkgs/applications/science/electronics/librepcb/default.nix:11]:
```suggestion
    rev = version;
```

**Review by prusnak — CHANGES_REQUESTED** [review]
The project uses tags, so we can leverage this by fetching the tag instead of the commit hash. See suggestion below.

**Review by prusnak — APPROVED** [review]

**Comment by prusnak** [comment]
Please squash the second commit into the first one

**Comment by Luz** [comment]
reopened a fresh one, hope thats ok...
https://github.com/NixOS/nixpkgs/pull/102954

**Pull request closed without merging**